docker-compose.yaml

```diff
@@ -0,0 +1,33 @@
+services:
+  cinema:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    env_file:
+      - .env
+    ports:
+      - "8001:8000"
+    volumes:
+      - ./:/app

```

Comment:
Mounting the entire current directory into the container can lead to potential issues, such as including unnecessary files. Ensure your `.dockerignore` file excludes files that are not needed in the container.